deploy/redis-cluster/values.yaml

```diff
@@ -41,4 +41,9 @@ sentinel:
   cpu: 0.2
   memory: 0.2
   storage: 5
+  replicas: 3
+
+proxy:
+  cpu: 2
+  memory: 6
```

Comment:
The default cpu and memory values are too large. It is recommended to change them to smaller values, such as 0.5, so that scenarios like local testing can create Twemproxy instances without resource constraints.